### PR TITLE
[Translation zh-cn] updates > 2017 > 09 > picture-in-picture.md

### PR DESCRIPTION
**Pull request by limhenry**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-CN/review/369)

### CODE REVIEW

**Comment by sotayamashita** [comment]
It fails because of `Error: OAuth2 authentication requires a token or key & secret to be set`

```
[10:06:37] Starting 'test:travis-init'...

[10:06:37] 'test:travis-init' errored after 28 ms

[10:06:37] Error: OAuth2 authentication requires a token or key & secret to be set

    at __dirname.authenticate (/home/travis/build/google/WebFundamentals/node_modules/github/lib/index.js:359:23)

    at Gulp.<anonymous> (/home/travis/build/google/WebFundamentals/gulp-tasks/test.js:1305:10)

    at module.exports (/home/travis/build/google/WebFundamentals/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)

    at Gulp.Orchestrator._runTask (/home/travis/build/google/WebFundamentals/node_modules/gulp/node_modules/orchestrator/index.js:273:3)

    at Gulp.Orchestrator._runStep (/home/travis/build/google/WebFundamentals/node_modules/gulp/node_modules/orchestrator/index.js:214:10)

    at Gulp.Orchestrator.start (/home/travis/build/google/WebFundamentals/node_modules/gulp/node_modules/orchestrator/index.js:134:8)

    at /home/travis/build/google/WebFundamentals/node_modules/gulp/bin/gulp.js:129:20

    at nextTickCallbackWith0Args (node.js:489:9)

    at process._tickCallback (node.js:418:13)

    at Function.Module.runMain (module.js:443:11)
```

**Comment by petele** [comment]
if you sync with the latest from master, I checked in a fix for that earlier today.

File: src/content/zh-cn/updates/2017/09/picture-in-picture.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: 让我们来讨论如何使用一个Web API以在桌面创建一个浮动视频窗口。 #}
+{# wf_blink_components: Blink>Media #}
+
+# 画中画 (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Translated by
```
Comment (petele) [inline src/content/zh-cn/updates/2017/09/picture-in-picture.md]: we've typically been asking people to put the translation credit at the bottom of the page.

File: src/content/zh-cn/updates/2017/09/picture-in-picture.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+{% include "web/_shared/contributors/henrylim.html" %}
+
+自2017年4月起，[Android O的Chrome支持画中画](https://developer.android.com/about/versions/oreo/android-8.0.html#opip)。这功能可让用户在同个时间，在悬浮窗中的`<video>`元素播放视频，一边做其他的事情。
```
Comment (petele) [inline src/content/zh-cn/updates/2017/09/picture-in-picture.md]: please wrap lines at 80 chars

File: src/content/zh-cn/updates/2017/09/picture-in-picture.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      document.exitPictureInPicture()
+      .catch(error => {
+        // 视频无法退出画中画模式
+      });
+    }
+  });
+</script>
+```
+
+注意: 目前没有任何的游览器可以运行以上的代码。
```
Comment (petele) [inline src/content/zh-cn/updates/2017/09/picture-in-picture.md]: [Callouts](https://developers.google.com/web/resources/markdown-syntax#callouts) like `Note:` should NOT be translated, they're handled differently by the markdown processor.

**Pull request merged**